src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -138,7 +154,8 @@ private enum Mode
         [HtmlAttributeName(FallbackTestValueAttributeName)]
         public string FallbackTestValue { get; set; }
 
-        // Properties are protected to ensure subclasses are correctly activated. Internal for ease of use when testing.
+        // Properties are protected to ensure subclasses are correctly activated.
+        // Internal for ease of use when testing.
```

Comment:
nit: why did this change? don't churn for churn's sake or due to a personal preference